onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -852,6 +842,7 @@ void addGlobalMethods(py::module& m, const Environment& env) {
   });
 #endif
 
+#if 0  // TODO: RyanHill - Add interface to shared providers so that this information can be queried
```

Comment:
can this be addressed in this PR? we don't want to break any api's.

Reply:
Yeah, it'll just take more time. Looking at it closer, I should probably scope it so that it returns an empty vector of strings vs removing all of the OpenVINO functions.

I'll try getting it to work tomorrow.

Reply:
Forgot to resolve, I did do it what was then tomorrow.